internal/files/files.go

```diff
@@ -19,3 +19,18 @@ func NewSHA256(filename string) (string, error) {
 	}
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
+
+func SaveToTempFile(closer io.ReadCloser) (string, error) {
```

Comment:
Would be good to add a doc that discloses that it's the callers responsibility to clean up temp files.

Reply:
Sure!

Reply:
Ah..